js/stores/appStore.js

```diff
@@ -122,6 +122,9 @@ const emitChanges = debounce(appStore.emitChanges.bind(appStore), 5)
  */
 function handleChangeSettingAction (state, settingKey, settingValue) {
   switch (settingKey) {
+    case settings.HARDWARE_ACCELERATION_ENABLED:
+      app.setBooleanPref('hardware_acceleration_mode.enabled', settingValue)
```

Comment:
should probably call this in a setImmediate block

Reply:
agree but it doesn't change frequently so not going to spend time on it and retest for now.